packages/cli/__tests__/zosjobs/__system__/submit/uss-file/__scripts__/submit_valid_uss_file_fully_qualified.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+JCL=$1
+HOST=$2
+PORT=$3
+USER=$4
+PASS=$5
+
+# pass the uss file name as an argument to the script
+zowe zos-jobs submit uss-file $JCL --host $HOST --port $PORT --user $USER --password $PASS --ru=false
```

Comment:
I like that we use all sort of combinations of aliases for out tests 😋 

sometimes `--response-format-json`, `--rfj`, `--ru=false`, `--no-ru` 😋